mithril-common/src/chain_observer/model.rs

```diff
@@ -1,12 +1,42 @@
 use anyhow::anyhow;
-use serde::Serialize;
+use pallas_codec::minicbor::{decode, Decode, Decoder};
+use pallas_primitives::{conway::PlutusData, ToCanonicalJson};
```

Comment:
Do we want to target specifically `conway` era?

Reply:
this is actually the plutus structure which doesnt really depends on the era, it has its own version and we follow the latest available.

The plutus contract for inline datum follows the same for any era by now.

The import is needed from premitives to avoid repeating ourselves.

The root of this import should be alonzo tho, where it is really implemented by pallas-primitives. Just have updated it.

Reply:
reference: https://github.com/IntersectMBO/plutus/blob/master/plutus-core/plutus-core/src/PlutusCore/Data.hs